upgrade/Stage_0_Prerequisites.md

```diff
@@ -159,6 +179,15 @@ after a break, always be sure that a typescript is running before proceeding.
    ENDPOINT=https://put.the/url/here/
    ```
 
+1. This step should ONLY be performed if an http proxy is required to access a public endpoint on the internet for the purpose of downloading artifacts.
+CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints.
+The http proxy variables must be `unset` after the desired artifacts are downloaded. Failure to unset the http proxy variables after downloading artifacts will cause many failures in subsequent steps.
+
+   ```bash
+   export https_proxy=https://example.proxy.net:443
+   export http_proxy=http://example.proxy.net:80
```

Comment:
The http_proxy does make sense in this case because they may provide an --endpoint argument that is an HTTP URL.